mitodl/edx-username-changer
# Add Lint/Testing framework

#### What are the relevant tickets?
https://github.com/mitodl/edx-username-changer/issues/2

#### What's this PR do?
- Updates the configurations through `edx-cookiecutter`
- Linting and testing is added

#### How should this be manually tested?
- Check that CI is working 
- Check the pylint/pytest related CI results
- Check that the functionality of the plugin is working fine
